sanic/blueprint_group.py

```diff
@@ -121,7 +160,30 @@ def __len__(self) -> int:
         """
         return len(self._blueprints)
 
-    def insert(self, index: int, item: object) -> None:
+    def _sanitize_blueprint(self, bp: "sanic.Blueprint") -> "sanic.Blueprint":
+        """
+        Sanitize the Blueprint Entity to override the Version and strict slash behaviors as required.
+
+        :param bp: Sanic Blueprint entity Object
+        :return: Modified Blueprint
+        """
+        item_prefix = bp.url_prefix or ""
+        bp.url_prefix = f"{self._url_prefix}{item_prefix}"
```

Comment:
Doesn't this have the potential of `/foo//bar` or `/foobar`, instead of the intended: `/foo/bar`?